tests/test_helpers/click_helpers.py

```diff
@@ -1,9 +1,3 @@
-# This is copied and slightly modified from the acryl-datahub project
```

Comment:
why are you removing the license headers? these should not be removed from the files that have them